eng/pipelines/templates/jobs/archetype-sdk-mgmt.yml

```
@@ -5,6 +5,12 @@ jobs:
     steps:
       #- script: "echo $(system.pullrequest.pullrequestnumber), https://github.com/$(build.repository.id), https://github.com/$(build.repository.ID)"
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
+      - script: |
```

```suggestion
      - pwsh: |
```

You can directly use a pwsh task instead of calling pwsh manually